src/shared/auth/wsClient.ts

```diff
@@ -107,12 +104,15 @@ export const createAuthServerWsClient = (authServerUrl?: string) => {
 
       return response
     } catch (e) {
-      console.error('Error recovering request', e)
-      captureException(e)
+      handleError(e, 'Error recovering request', {
+        trackingData: {
+          browserTime: Date.now(),
+          requestType: response?.method ?? 'Unknown'
```

Comment:
These two properties seem to be related to the analytics tracking down below, is that correct?